controller/api/util/api_utils.go

```diff
@@ -100,3 +102,46 @@ func BuildStatSummaryRequest(p StatSummaryRequestParams) (*pb.StatSummaryRequest
 
 	return statRequest, nil
 }
+
+// BuildResource parses input strings, typically from CLI flags, to build a
+// Resource object for use in the Conduit Public API.
+func BuildResource(namespace string, args ...string) (pb.Resource, error) {
+	switch len(args) {
+	case 0:
+		return pb.Resource{}, errors.New("No resource arguments provided")
```

Comment:
aiui, it's idiomatic to return nil instead of pb.Resource{} in these cases (since the value cannot be used when err != nil)